lisp/ui/widgets/dbmeter.py

```diff
@@ -77,6 +87,35 @@ def plot(self, peaks, _, decayPeak):
 
         self.update()
 
+    def updateMarkings(self):
+        self._markings = []
+
+        height = self.height()
+        font_height = QFontMetrics(self.font()).height()
+        steps = [1, 5, 10, 25, 50, 100, 250]
```

Comment:
```suggestion
        steps = [2, 4, 8, 16, 32, 64, 128]
```
I found that this scale is a bit more pleasing, but le me know what do you think :)

Reply:
The scale given is designed to give values that one might expect to find on, say, the meter-bridge of a sound desk.

I'll grant you, that some revision was needed: a gap of 100 dB was implausible.